src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public class Customer {
```

Comment:
Do you intend to have classes derived from `Customer`? If not, the class needs to be made `final`.